packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -6,9 +6,38 @@ import "../../core/IDAO.sol";
 
 /// @title IMajorityVoting
 /// @author Aragon Association - 2022
-/// @notice The interface for majority voting contracts. We use the following definitions:
-///     Relative support: `N_yes / (N_yes + N_no)`
-///     Total support   : `N_yes / N_total`
+/// @notice The interface of majority voting plugin.
+///
+///  #### Parameterization
+///  We define two parameters
+///  $$\texttt{support} = \frac{N_\text{yes}}{N_\text{yes}+N_\text{no}}$$
+///  and
+///  $$\texttt{participation} = \frac{N_\text{yes}+N_\text{no}+N_\text{abstain}}{N_\text{total}}$$
+///  where $N_\text{yes}$, $N_\text{no}$, and $N_\text{abstain}$ are the yes, no, and abstain votes that have been casted and $N_\text{total}$ is the total voting power available at proposal creation time.
+///  Majority voting implies that the support threshold is set with
+///  $$\texttt{supportThreshold} \ge 50\% .$$
```

Comment:
There is comments like this repeating elsewhere, but i leave my opinion only here:

This text is not human readable, IMO we should have an explanation comment such as: https://github.com/bitcoin/bitcoin/blob/master/src/consensus/merkle.cpp#L8

or another option could be to link an **.md file here instead of these type of comments

Reply:
> or another option could be to link an **.md file here instead of these type of comments

Try out `yarn docgen` to see the outcome.
This generates the docs and there it becomes much more readable.
Linking to another `.md` is not supported.

NatSpec is based on doxygen, which supports writing formulas: https://doxygen.nl/manual/formulas.html

If we display the formulas inline and unformatted, they become very lengthy, and we lose clarity.
Let's be happy for now that we finally have good documentation of what the code is doing for what reason and that it is properly displayed in the developer portal.

The question about where the MajorityVoting documentation will be found is another topic that needs to be addressed.